TLM/TLM/Util/Record/SpeedLimitLaneRecord.cs

```diff
@@ -43,16 +49,17 @@ public class SpeedLimitLaneRecord : IRecordable {
                 LANE_TYPES,
                 VEHICLE_TYPES,
                 sort: false);
-            foreach(var lane in lanes) {
+            foreach (var lane in lanes) {
                 SpeedLimitLaneRecord laneData = new SpeedLimitLaneRecord {
                     LaneId = lane.laneId,
                     LaneIndex = lane.laneIndex,
-                    SegmentId = segmentId,
-                    LaneInfo = segmentId.ToSegment().Info.m_lanes[lane.laneIndex],
                 };
                 ret.Add(laneData);
```

Comment:
Same thing here; if we know list size in advance it is worth initialising the list with the required capacity to reduce memalloc calls.